packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -1408,12 +1408,24 @@ export default async function initSanity(
       .clone()
       .config({apiVersion: 'v2021-06-07'})
 
-    const grants = await client.request({uri: `organizations/${orgId}/grants`})
-    const group: {grants: {name: string}[]}[] = grants[requiredGrantGroup] || []
-    return group.some(
-      (resource) =>
-        resource.grants && resource.grants.some((grant) => grant.name === requiredGrant),
-    )
+    try {
+      const grants = await client.request({uri: `organizations/${orgId}/grants`})
+      const group: {grants: {name: string}[]}[] = grants[requiredGrantGroup] || []
+      return group.some(
+        (resource) =>
+          resource.grants && resource.grants.some((grant) => grant.name === requiredGrant),
+      )
+    } catch (err) {
+      // If we get a 401, it means we don't have access to this organization
+      // probably because of implicit membership
+      if (err.statusCode === 401) {
+        debug('No access to organization %s (401)', orgId)
+        return false
+      }
+      // For other errors, log them but still return false
```

Comment:
Linting requires us to return a value from all branches of this function and I figured it was safe to silently ignore other kinds of errors, but let me know if you'd prefer to re-throw here.